DOCUMENTATION.md

```diff
@@ -54,70 +54,65 @@ Electron is a framework that uses web technologies (HTML, CSS, and JS) to build
 
 ### Linux
 
-* **Python** version 2.7.x. It is recommended to check your Python version since some distributions like CentOS 6.x still use Python 2.6.x.
+- **Python** version 2.7.x. It is recommended to check your Python version since some distributions like CentOS 6.x still use Python 2.6.x.
```

Comment:
Is there a particular reason to change this `-` to `*`?

Reply:
MD Linter was complaining  about mixed usage for unordered lists. I changed these to keep the consistency across all lists. Since I am using a MD Table of Content Generator tool, it uses `-` so it forced me to use `-` for all the other lists.